### PR TITLE
Fix export for relation content

### PR DESCRIPTION
**Pull request by ArnoEgli**
In relation with: https://github.com/araise-dev/CrudBundle/pull/52

### CODE REVIEW

File: src/Resources/views/tailwind_2/_header.html.twig [hunk 1]
```diff
@@ -1,6 +1,5 @@
 {# Table Header #}
-{% if table.option('title') or view is defined and view.definition.hasCapability(constant('araise\\CrudBundle\\Enums\\Page::EXPORT')) or table.filterExtension and table.filterExtension.filters|length > 0 or table.searchExtension and table.option('searchable') %}
-
+{% if table.option('title') or view is defined and table.option('definition').hasCapability(constant('araise\\CrudBundle\\Enums\\Page::EXPORT')) or table.filterExtension and table.filterExtension.filters|length > 0 or table.searchExtension and table.option('searchable') %}
```
Comment (tuxes3) [inline src/Resources/views/tailwind_2/_header.html.twig:2]: please group the `and` and `or` in brackets for better readability

File: src/Resources/views/tailwind_2/_header.html.twig [hunk 2]
```diff
@@ -52,6 +51,12 @@
                             <p class="whatwedo-utility-bold whitespace-nowrap mb-3 mt-3">{{ 'araise_table.download.info' | trans }}</p>
                             <div>
                                 {% for label,queryParameters in {'araise_table.page' : app.request.query.all, 'araise_table.all' : {'all':1}|merge(app.request.query.all)} %}
+                                    {% if view.definition != table.option('definition') %}
```
Comment (tuxes3) [inline src/Resources/views/tailwind_2/_header.html.twig:54]: NIT: has this been tested with a relation content of the same definition?

**Review by tuxes3 — CHANGES_REQUESTED** [review]

File: src/Resources/views/tailwind_2/_header.html.twig (hunk 2 above)
Comment (ArnoEgli) [reply src/Resources/views/tailwind_2/_header.html.twig]: Good catch, this check is unnecessary. I have removed the check and tested it with an entity category with under categories.

File: src/Resources/views/tailwind_2/_header.html.twig (hunk 1 above)
Comment (ArnoEgli) [reply src/Resources/views/tailwind_2/_header.html.twig]: It's now more readable

**Review by tuxes3 — APPROVED** [review]

**Review by tuxes3 — APPROVED** [review]

**Pull request merged**